packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -1838,3 +1838,33 @@
 .cfd-accounts__compare-table-title {
     cursor: pointer;
 }
+.acuity {
+    &-container {
+        background: var(--general-section-3);
+        display: flex;
+        position: relative;
+        top: 2.3rem;
+        margin-bottom: 2.7rem;
+        align-items: center;
+    }
+    &-banner {
+        display: flex;
```

Comment:
In my opinion you don't have to define display:flex for every wrapper but if it doesn't work then it's fine.

Reply:
Resolved